ui/checkScript.js

```diff
@@ -0,0 +1,52 @@
+const fs = require("fs");
```

Comment:
This is way overcomplicated in node.js/JavaScript, and I'd definitely encourage simply using an npm script for this. It's so much easier.